docs/docs/References/Version_Compatibility.md

```diff
@@ -11,8 +11,6 @@
 | [3.0.0](../Release_Notes/3.0.0/3.0.0.md) |            6.0.6.0            | [Latest Release](https://cd.delphix.com/docs/latest/new-features) |
 | [2.1.0](../Release_Notes/2.1.0/2.1.0.md) |            6.0.3.0            | [Latest Release](https://cd.delphix.com/docs/latest/new-features) |
 | [2.0.0](../Release_Notes/2.0.0/2.0.0.md) |            6.0.2.0            | [Latest Release](https://cd.delphix.com/docs/latest/new-features) |
-| [1.0.0](../Release_Notes/1.0.0/1.0.0.md) |            6.0.2.0            | [Latest Release](https://cd.delphix.com/docs/latest/new-features) |
```

Comment:
Instead of removing the compatibility, we can limit the version from 6.0.2.0 to 14.0.0.0

Reply:
updated. Changes can be seen here :
https://rakshandabhat.github.io/virtualization-sdk/References/Version_Compatibility/